src/dev-utils/view.js

```diff
@@ -550,7 +550,7 @@ class RangeParser {
 			if ( item.bracket == ELEMENT_RANGE_START_TOKEN || item.bracket == TEXT_RANGE_START_TOKEN ) {
```

Comment:
Maybe this part of the code can be refactored to store range start and range end, instead of creating a range only to modify it later.